subsys/bluetooth/controller/Kconfig

```diff
@@ -1001,6 +1004,13 @@ config BT_CTLR_HCI_CODEC_AND_DELAY_INFO
 	  Enable HCI commands to read information about supported
 	  codecs, codec capabilities, and controller delay.
 
+config BT_CTLR_SUBRATING
+	bool "Subrate Connection commands"
```

Comment:
```suggestion
	bool "LE Connection Subrating"
```

Reply:
Agree, this enabled the whole feature, not just commands